src/SpineOpt.jl

```diff
@@ -40,151 +41,138 @@ include("run_spineopt.jl")
-	"constraints/constraint_operating_point_sum.jl",
-	"constraints/constraint_nodal_balance.jl",
-	"constraints/constraint_node_injection.jl",
-	"constraints/constraint_node_state_capacity.jl",
-	"constraints/constraint_cyclic_node_state.jl",
-	"constraints/constraint_ratio_unit_flow.jl",
-	"constraints/constraint_ratio_out_in_connection_flow.jl",
-	"constraints/constraint_ratio_out_in_connection_intact_flow.jl",
-	"constraints/constraint_connection_flow_capacity.jl",
-	"constraints/constraint_connection_intact_flow_capacity.jl",
-	"constraints/constraint_connection_flow_intact_flow.jl",
-	"constraints/constraint_connection_intact_flow_ptdf.jl",
-	"constraints/constraint_candidate_connection_flow_ub.jl",
-	"constraints/constraint_candidate_connection_flow_lb.jl",
-	"constraints/constraint_connection_flow_lodf.jl",
-	"constraints/constraint_connections_invested_available.jl",
-	"constraints/constraint_connections_invested_transition.jl",
-	"constraints/constraint_connection_lifetime.jl",
-	"constraints/constraint_storages_invested_available.jl",
-	"constraints/constraint_storages_invested_transition.jl",
-	"constraints/constraint_storage_lifetime.jl",
-	"constraints/constraint_units_on.jl",
-	"constraints/constraint_units_available.jl",
-	"constraints/constraint_minimum_operating_point.jl",
-	"constraints/constraint_min_up_time.jl",
-	"constraints/constraint_min_down_time.jl",
-	"constraints/constraint_unit_state_transition.jl",
-	"constraints/constraint_user_constraint.jl",
-	"constraints/constraint_units_invested_available.jl",
-	"constraints/constraint_units_invested_transition.jl",
-	"constraints/constraint_unit_lifetime.jl",
-	"constraints/constraint_split_ramps.jl",
-	"constraints/constraint_unit_pw_heat_rate.jl",
-	"constraints/constraint_ramp_up.jl",
-	"constraints/constraint_max_start_up_ramp.jl",
-	"constraints/constraint_min_start_up_ramp.jl",
-	"constraints/constraint_max_nonspin_ramp_up.jl",
-	"constraints/constraint_min_nonspin_ramp_up.jl",
-	"constraints/constraint_ramp_down.jl",
-	"constraints/constraint_max_shut_down_ramp.jl",
-	"constraints/constraint_min_shut_down_ramp.jl",
-	"constraints/constraint_max_nonspin_ramp_down.jl",
-	"constraints/constraint_min_nonspin_ramp_down.jl",
-	"constraints/constraint_res_minimum_node_state.jl",
-	"constraints/constraint_fix_node_pressure_point.jl",
-	"constraints/constraint_compression_ratio.jl",
-	"constraints/constraint_storage_line_pack.jl",
-	"constraints/constraint_connection_flow_gas_capacity.jl",
-	"constraints/constraint_max_node_pressure.jl",
-	"constraints/constraint_min_node_pressure.jl",
-	"constraints/constraint_max_node_voltage_angle.jl",
-	"constraints/constraint_min_node_voltage_angle.jl",
-	"constraints/constraint_node_voltage_angle.jl",
-	"constraints/constraint_connection_unitary_gas_flow.jl",
-	"constraints/constraint_mp_any_invested_cuts.jl",
-]
+include("run_spineopt_sp.jl")
+include("run_spineopt_benders_algorithm.jl")
```

Comment:
If we're renaming files, then perhaps we should go for more consistency - perhaps:

```
run_spineopt_standard.jl
run_spineopt_benders.jl
run_spineopt_mga.jl

```

Also, till now we have filenames in lower case only - my preference would be to preserve this.